server/tests/api_v5/anonymous_account/test_account_send_email_validation.py

```diff
@@ -0,0 +1,37 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+
+from parsec._parsec import anonymous_account_cmds
+from tests.common import Backend, HttpCommonErrorsTester
+from tests.common.client import AccountRpcClient
+
+
```

Comment:
You should add a `test_anonymous_account_account_send_email_validation_token_send_email_bad_outcome` working similarly to the existing test `test_authenticated_invite_new_device_send_email_bad_outcome` to test what happen if the email cannot be send